test/e2e/page-objects/pages/confirmations/redesign/transaction-confirmation.ts

```diff
@@ -223,26 +236,18 @@ class TransactionConfirmation extends Confirmation {
   }
 
   /**
-   * Checks if the sender account is displayed in the transaction confirmation page.
+   * Checks that the sender account is displayed on the transaction confirmation page.
    *
    * @param account - The sender account to check.
    */
-  async checkIsSenderAccountDisplayed(account: string): Promise<boolean> {
+  async checkSenderAccountIsDisplayed(account: string): Promise<void> {
```

Comment:
the original method was called without asserting the return value in code base, making it effectively do nothing. I updated the implementation to prevent false positives.